### PR TITLE
Add aniline sp

### PR DESCRIPTION
**Pull request by amcisaac**
## New Submission Checklist

- [x] Created a new folder in the submissions directory containing the dataset
- [x] Added `README.md` describing the dataset see [here](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2020-03-26-OpenFF-Gen-2-Torsion-Set-6-supplemental-2) for examples
- [x] All files used to produce the dataset are included with a description
- [x] Dataset follows the QCSubmit schema defined for [Datasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L340), [OptimizationDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1062) and [TorsionDriveDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1225)
- [x] Dataset filename matches pattern `dataset*.json`; may feature a compression extension, such as `.bz2`
- [x] A PDF depicting the molecules is attached, in the case of torsiondrives this should include the highlighting of the central bond, this can be done automatically using [qcsubmit](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L854). 
- [x] QCSubmit validation passed
- [x] Made a new dataset entry in the mapping table in repository `README.md`
- [x] Ready to submit!

### CODE REVIEW

**Review by ntBre — COMMENTED** [review]
This looks reasonable to me, but is there a reason not to use `OptimizationResultCollection.create_basic_dataset` instead of doing something very similar manually? I know I did my last one manually, but I think the built-in form should be preferred where possible.

**Comment by amcisaac** [comment]
Hm, I just copied one of our old PR's as a template, and it was done this way. I'll give the `to_basic_dataset` a try!

**Comment by amcisaac** [comment]
Seems to have produced the same result, but it's much faster! Thanks for the suggestion.

File: README.md [hunk 1]
```diff
@@ -213,6 +213,7 @@ These are currently used to compute properties of a minimum energy conformation
...
 |`OpenFF NAGL2 ESP Timing Benchmark v1.1` | [2024-09-18-OpenFF-NAGL2-ESP-Timing-Benchmark-v1.1](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-09-18-OpenFF-NAGL2-ESP-Timing-Benchmark-v1.1) | Single point ESP calculations for timing/memory benchmarking | 'P', 'S', 'N', 'C', 'Cl', 'F', 'Br', 'O', 'H', 'I' | |
 |`OpenFF Sulfur Hessian Training Coverage Supplement v1.0` | [2024-09-18-OpenFF-Sulfur-Hessian-Training-Coverage-Supplement-v1.0](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-09-18-OpenFF-Sulfur-Hessian-Training-Coverage-Supplement-v1.0) | Additional Hessian training data for Sage sulfur and phosphorus parameters (from 'OpenFF Sulfur Optimization Training Coverage Supplement v1.0') | O, S, C, Cl, P, N, F, Br, H | |
+| `OpenFF Aniline Para Hessian v1.0` | [2024-10-07-OpenFF-Aniline-Para-Hessian-v1.0](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-10-07-OpenFF-Aniline-Para-Hessian-v1.0) | Hessian single points for the final molecules in the `OpenFF Aniline Para Opt v1.0 dataset` | 'O', 'Cl', 'S', 'Br', 'H', 'F', 'N', 'C' ||
```
Comment (lilyminium) [inline README.md:216]:
```suggestion
| `OpenFF Aniline Para Hessian v1.0` | [2024-10-07-OpenFF-Aniline-Para-Hessian-v1.0](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-10-07-OpenFF-Aniline-Para-Hessian-v1.0) | Hessian single points for the final molecules in the `OpenFF Aniline Para Opt v1.0` dataset | 'O', 'Cl', 'S', 'Br', 'H', 'F', 'N', 'C' ||
```

(just to be super clear what the dataset is called!)

**Review by lilyminium — APPROVED** [review]
LGTM with just a minor nitpick in the readme, thanks for putting it up and @ntBre for the review!!

**Pull request merged**